### PR TITLE
PT-3725

### PR DESCRIPTION
**Pull request by akak1977**
## Description
PT-3725
### Jira-link:
https://virtocommerce.atlassian.net/browse/PT-3724

### CODE REVIEW

File: src/VirtoCommerce.Platform.Core/Assets/BasicBlobProvider.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Linq;
+using Microsoft.Extensions.Options;
+using VirtoCommerce.Platform.Core.Settings;
+
+
+namespace VirtoCommerce.Platform.Core.Assets
+{
+    public abstract class BasicBlobProvider
```
Comment (ksavosteev) [inline src/VirtoCommerce.Platform.Core/Assets/BasicBlobProvider.cs:9]: Personally I'd extracted this into a new service instead of an abstract class.

**Review by ksavosteev — APPROVED** [review]

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/assets/blades/asset-upload.js [hunk 2]
```diff
@@ -64,15 +64,12 @@ angular.module('platformWebApp')
...
-                    refreshParentBlade();
-                    blade.uploadCompleted = true;
-                };
-
                 uploader.onSuccessItem = function (fileItem, images) {
+                    refreshParentBlade();
                     if (blade.onUploadComplete) {
                         blade.onUploadComplete(images);
                     }
+                    blade.uploadCompleted = true;
```
Comment (eMazeika) [inline src/VirtoCommerce.Platform.Web/wwwroot/js/app/assets/blades/asset-upload.js:72]: why this change was needed? Comment in code, pls

Comment (akak1977) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/assets/blades/asset-upload.js:72]: This is a bug. As I saw onCompleteAll always called. Therefore no chance to see an error. The user always sees the message about successful upload.
